sensors/cartesian_sensor/cartesian_sensor.ino

```diff
@@ -9,3 +9,27 @@ Cartesian_Sensor::Cartesian_Sensor(const int &pin): Sensor(&pin) {
 double Cartesian_Sensor::compute_magnitude(double x, double y, double z) {
   return sqrt(pow(x, 2) + pow(y, 2) + pow(z, 2));
 }
+
+double Cartesian_Sensor::get_raw_x() {
+  return m_raw_x;
+}
+
+double Cartesian_Sensor::get_raw_y() {
+  return m_raw_y;
+}
+
+double Cartesian_Sensor::get_raw_z() {
+  return m_raw_z;
+}
```

Comment:
Will implement this next sprint!

Reply:
Question: wouldn't this require the struct to be declared in the scope of allsensors.ino (the main code that runs on the car) as well, given that it would be our intended return value?

Reply:
Since it's a pretty minimal struct, you can declare it at the top of `cartesian_sensor.h`. That way, whenever a consumer uses a cartesian sensor (and thus `#include`s its header file), the declaration will already be available to use. Does that make sense?

Reply:
Done!